### ISSUE TITLE
Why is Graphviz being installed?

### ISSUE THREAD

**Issue by akademy** [DiamondLightSource/python-copier-template#220]
Why is graphviz defined in the template? Seems like this will be unnecessary in most pips.

https://github.com/DiamondLightSource/python-copier-template/blob/2d761101ae3c273475ffd21f74a337a3304af637/template/Dockerfile.jinja#L8

It's probably being installed by many projects for no reason.

**Comment by coretl** [comment]
It's there because the default api docs used to include a class hierarchy which required graphviz, however although the directive is included:
https://github.com/DiamondLightSource/python-copier-template/blob/2d761101ae3c273475ffd21f74a337a3304af637/template/%7B%25%20if%20sphinx%20%25%7Ddocs%7B%25%20endif%20%25%7D/conf.py.jinja#L45
It doesn't appear to be used in the docs at all. Maybe we should remove both the directive and the apt install and make a how-to telling you how to put it back in?

**Comment by akademy** [comment]
Yeah, sounds sensible (can always put it back again if it causes a problem).

I guess we'll still need to include the `apt-get update` though, as some builds may assume it's already run? (Or perhaps no‑one retrospectively copies the template...)
